added UITextView and test coverage for UITextView

andreHelsinki opened the pull request:
This commit includes an extension for ConfigurationSet for Base: UITextView and majority of the tests. Two test are not implemented in a clever way, i use workaround (please check the bottom of UITextViewSpecs for more details). In addition I am not sure how to test function scrollRangeToVisible. Delegate methods are not implemented at the moment.

PSchu committed:
updatedReadme

Commit:
Merge pull request #5 from imagineon/release/v0.3.0

Release 0.3.0

PSchu committed:
updated Podspec

Commit:
added UITextView and test coverage for UITextView

PSchu commented on Tests/UITextViewSpecs.swift:
```diff
@@ -0,0 +1,182 @@
+class UITextViewSpecs: QuickSpec {
+    
+    override func spec() {
+        it("can set text") {
```
I would envelop all tests in a describe function `describe("UIViewConfigurator") {` this will improve the test names, to read like a sentence.

PSchu commented on Tests/UITextViewSpecs.swift:
```diff
@@ -0,0 +1,182 @@
+        
+        it("can set isSelectable") {
+            let selectable = true
+            let testView: UITextView = .build { set in
```
The default Value for Selectable is true, therefor this test would also succeed if nothing would be configured

PSchu commented on Sources/UITextView.swift:
```diff
@@ -0,0 +1,139 @@
+        })
+    }
+    
+    func scrollRangeToVisible(_ range: NSRange) -> ConfigurationSet<Base> {
```
I think this function is not needed during View configuration, only later in its life cycle.

PSchu commented on Sources/UITextView.swift:
```diff
@@ -0,0 +1,139 @@
+
+
+public extension ConfigurationSet where Base: UITextView {
+    
```
I would add a configuration Method for the Delegate.

PSchu reviewed (commented):
I looked over your pull request and made some comments. I will look into the Test Problem.

Commit:
Implementation of Peter's comments

added delegate method to UITextViewConfigurator
added test for delegate method
changed tests for UITextView.typingAttrbutes and .linkTextAttributes
added describe("UITextViewConfigurator") and in the test coverage UITextViewSpecs
removed scrollToVisible in UITextViewConfigurator
changed test for UITestView.isSelectable

Commit:
removed setSomething

Commit:
Merge branch 'develop' into master

PSchu commented:
With Pull Request #9 this got obsolete.
